pages/dkp/kommander/2.2/legal/version-policy/index.md

```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```

Comment:
```suggestion
Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```

Reply:
Not sure if Preview should be capitalized here and below as other references are on the page?  Not title, but other references.